test/TestProjects/FirstTest-Cadl/Generated/Demo2Client.cs

```diff
@@ -167,15 +182,17 @@ public virtual Response<Thing> HelloAgain(RoundTripModel action, CancellationTok
         /// </code>
         /// 
         /// </remarks>
-        public virtual async Task<Response> HelloAgainAsync(RequestContent content, RequestContext context = null)
+        public virtual async Task<Response> HelloAgainAsync(string p2, string p1, RequestContent content, ContentType contentType, RequestContext context = null)
```

Comment:
Where do we define `ContentType`? In Azure.Core or Models?

Reply:
In Azure.Core

Reply:
I think we need to discuss it because HLC is using the generated `ContentType` model.

Reply:
In protocol method, it use Azure.Core.ContentType since there is no model class generated in DPG.